src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -565,19 +567,40 @@ public static IQueryable<Release> HydrateReleaseForReleaseViewModel(IQueryable<R
                 .Include(r => r.Type);
         }
 
-        private static IQueryable<Release> HydrateReleaseForAmendment(IQueryable<Release> values)
+        private async Task<Release> HydrateReleaseForAmendment(Release release)
         {
             // Require publication / release / contact / type graph to be able to work out:
             // If the release is the latest
             // The contact
             // The type
-            return values.Include(r => r.Publication)
-                .Include(r => r.Content)
-                .ThenInclude(c => c.ContentSection)
-                .ThenInclude(c => c.Content)
-                .Include(r => r.Updates)
-                .Include(r => r.ContentBlocks)
-                .ThenInclude(r => r.ContentBlock);
+            await _context.Entry(release)
+                .Reference(r => r.Publication)
+                .LoadAsync();
+            await _context.Entry(release)
+                .Collection(r => r.Content)
+                .LoadAsync();
+            await release.Content.ForEachAsync(async cs =>
+            {
+                await _context.Entry(cs)
+                    .Reference(rcs => rcs.ContentSection)
+                    .LoadAsync();
+                await _context.Entry(cs.ContentSection)
+                    .Collection(s => s.Content)
+                    .LoadAsync();
+            });
+            await _context.Entry(release)
+                .Collection(r => r.Updates)
+                .LoadAsync();
+            await _context.Entry(release)
+                .Collection(r => r.ContentBlocks)
+                .LoadAsync();
+            await release.ContentBlocks.ForEachAsync(async rcb =>
+            {
```

Comment:
Again, I don't think you need the curly braces so it's just a question of personal taste.